roles/openshift_health_checker/test/docker_storage_test.py

```diff
@@ -0,0 +1,214 @@
+import pytest
+
+
+from openshift_checks import OpenShiftCheckException
+from openshift_checks.docker_storage import DockerStorage
+
+
+def dummy_check(execute_module=None):
+    def dummy_exec(self, status, task_vars):
+        raise Exception("dummy executor called")
+    return DockerStorage(execute_module=execute_module or dummy_exec)
+
+
+@pytest.mark.parametrize('is_containerized, group_names, is_active', [
+    (False, ["masters", "etcd"], False),
+    (False, ["masters", "nodes"], True),
+    (True, ["etcd"], True),
+])
+def test_is_active(is_containerized, group_names, is_active):
+    task_vars = dict(
+        openshift=dict(common=dict(is_containerized=is_containerized)),
+        group_names=group_names,
+    )
+    assert DockerStorage.is_active(task_vars=task_vars) == is_active
+
+
+non_atomic_tv = {"openshift": {"common": {"is_atomic": False}}}
```

Comment:
What does `tv` stand for?! Ah, task_vars!

Please let's use the full name. It's enough cognitive load to have to know what is "task_vars", let alone figure out `_tv` -> `_task_vars` on something that is used only once...